packages/encoding/karma.conf.cjs

```diff
@@ -15,7 +15,7 @@ module.exports = function (config) {
     frameworks: ["jasmine"],
 
     // list of files / patterns to load in the browser
-    files: ["dist/web/tests.js"],
+    files: ["build/web/tests.js"],
```

Comment:
I like this change btw. As this is a purely dev build artifact, we can bundle from `build/` to `build/web/` or something like that to free the "dist" name